plugin-jdbc/src/main/java/io/kestra/plugin/jdbc/SqlSplitter.java

```diff
@@ -0,0 +1,275 @@
+package io.kestra.plugin.jdbc;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+public class SqlSplitter {
+
+    public static String[] getQueries(String sql) {
```

Comment:
Backtick-quoted identifiers are supported by **MySQL, MariaDB, Snowflake, and BigQuery**, e.g.:

```sql
SELECT `id;hello`
```

SQL Server uses square brackets

```sql
SELECT [id;hello]
```

However, we might do this as follow up?

Reply:
Hmm let me check this in a dedicated unit test ;-)